package.json

```diff
@@ -26,7 +26,9 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "eslint": "^8.6.0",
-    "rollup": "^2.20.0"
+    "rollup": "^2.20.0",
```

Comment:
We'll need mocha here too

Reply:
Yes, you're right. I've installed globally on my machine